src/deployment/deploy.py

```diff
@@ -310,15 +308,13 @@ def get_instance_urls(self) -> List[str]:
                 "https://%s/%s" % (self.multi_tenant_domain, name)
                 for name in [
                     self.application_name,
-                    self.application_name + DOTNET_APPLICATION_SUFFIX,
                 ]
             ]
         else:
             return [
                 "https://%s.azurewebsites.net" % name
                 for name in [
```

Comment:
```suggestion
        if self.multi_tenant_domain:
            return "https://%s/%s" % (self.multi_tenant_domain, self.application_name)
        else:
            return "https://%s.azurewebsites.net" % self.application_name
```

Reply:
I mixed up the line number here, but these two blocks can be condensed into just the returns